libs/langchain/langchain/retrievers/sec_filings.py

```diff
@@ -0,0 +1,51 @@
+from __future__ import annotations
+
+from typing import Any, List
+
+from langchain.callbacks.manager import CallbackManagerForRetrieverRun
+from langchain.schema import BaseRetriever, Document
+
+
+class SecFilingsRetriever(BaseRetriever):
```

Comment:
if this is just an instance of KayAiRetriever with specific init params, would prefer not to add a separate class for it but we can keep the example in docs!